example/disasm/full.py

```diff
@@ -121,27 +123,28 @@
         if ad in done:
             continue
         done.add(ad)
-        allblocks = mdis.dis_multiblock(ad)
+        asmcfg = mdis.dis_multiblock(ad)
 
         log.info('func ok %.16x (%d)' % (ad, len(all_funcs)))
 
         all_funcs.add(ad)
-        all_funcs_blocks[ad] = allblocks
-        for block in allblocks:
+        all_funcs_blocks[ad] = asmcfg
```

Comment:
Given its name, `all_funcs_blocks` could contains only `asmcfg.blocks`.
This patch can also be written as `allblocks = mdis.dis_multiblock(ad).blocks`, the rest remaining the same

Reply:
This comment does not longer stand, please see below